math/kalc/Portfile

```diff
@@ -40,6 +39,8 @@ destroot {
         ${worksrcpath}/target/[cargo.rust_platform]/release/${name} \
         ${destroot}${prefix}/bin/
 
+    system -W ${worksrcpath} "./target/[cargo.rust_platform]/release/${name} --man > ${name}.1"
```

Comment:
Thanks @i0ntempest. Only nitpick is that if you're _calling_ the binary, that should ideally happen in the `post-build` phase, so that the only thing (ideally) happening in `destroot` is just copying files.

Reply:
Done